src/errors.rs

```diff
@@ -297,7 +297,7 @@ error_chain! {
         }
         ToolchainNotSelected {
             description("toolchain is not selected")
-            display("no override and no default toolchain set")
+            display("no override and no default toolchain set; run 'rustup default stable' to download and set the stable toolchain as default")
```

Comment:
lets not mention downloading here: it may already be present, or not, and its not the essence of the thing.